icefoganalytics/sfa-client
# Enhancement: Add Student CSFA Income and Expenses Sections to Student Application Endpoint Response

Relates to https://github.com/icefoganalytics/student-financial-aid/issues/2
Fixes https://github.com/icefoganalytics/sfa-client/issues/33

# Context

As part of the "view submitted applications and their current status" project, this task covers returning the student CSFA income and expenses data.

# Data Format

```js
{
  "id": "some-application-id",
  "csfaIncome": { "incomes": [] },
  "csfaExpenses": { "expenses": [] },
}
```

# Testing Instructions

1. Boot the `sfa-client` repo via `API_PORT=3100 dev up`
2. Boot the `student-financial-aid` front- and back-ends.
3. Log in to the app at http://localhost:8080/
4. Go to http://localhost:3000/api/portal/students/5/applications/3808 and you will see the empty CSFA income and CSFA expenses fields.
5. Go to http://localhost:3000/api/portal/students/124478/applications/30356 and you will see some CSFA income values.
6. Go to http://localhost:3000/api/portal/students/8634/applications/1995 and you will see some CSFA expense values.
